docs/changelogs/v0.14.md

```diff
@@ -57,3 +58,15 @@ the same binaries are still published under the old name at:
 
 - https://dist.ipfs.io/go-ipfs/
 - https://hub.docker.com/r/ipfs/go-ipfs/
+
+The libp2p identify useragent of Kubo has also been changed from `go-ipfs` to `kubo`.
+
+#### 🎒 `ipfs repo migrate`
+
+This new command allows the you to run the repo migration without starting the daemon.
+
+See `ipfs repo migrate --help` for more info.
+
+----
+
+[/ipfs/🚀🪐⭐💻😅❓💎🌈🌸🌚💰💍🌒😵🐶💁🤐🌎👼🙃🙅☺🌚😞🤤⭐🚀😃✈🌕😚🍻💜🐷⚽✌😊](https://ipfs.io/ipfs/🚀🪐⭐💻😅❓💎🌈🌸🌚💰💍🌒😵🐶💁🤐🌎👼🙃🙅☺🌚😞🤤⭐🚀😃✈🌕😚🍻💜🐷⚽✌😊)
```

Comment:
I've tested this work, if you try the link right now it will randomly work about 10% of the time (if it falls on a prerelease 0.14-rc1 bank).
Once they are updated they will all work.